crates/holochain_metrics/LICENSE-MIT

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The current policy is:

- If it's a core holochain crate dealing directly or indirectly with validation, it should be under the CAL.
- If it's a crate we may want other projects to use it should be either MIT or Apache-2.0 (or preferably both, since that is the de-facto standard for Rust crates), because that slightly lowers the barrier of usage for other projects.

I can add Apache-2.0 to this. That's the part we should try to clean up.

Reply:
Makes sense, and actually I think we've talked about this before but haven't got to the task yet. Thanks for reminding me!